build.sh

```diff
@@ -215,16 +202,7 @@ build_coreclr()
         echo "Failed to build coreclr components."
         exit 1
     fi
-}
-
-restoreBuildTools()
-{
-    echo "Restoring BuildTools..."
-    $__ProjectRoot/init-tools.sh
-    if [ $? -ne 0 ]; then
-        echo "Failed to restore BuildTools."
-        exit 1
-    fi
+	popd
```

Comment:
This "popd" command will not match the "pushd" command above if __SkipConfigure is set.